### PR TITLE
enable preview envs for blogpost

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Signed-off-by: Arsh Sharma <arshsharma461@gmail.com>

### CODE REVIEW

File: .github/workflows/preview.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+on: 
+  pull_request:
```
Comment (pchico83) [inline .github/workflows/preview.yaml:2]: should we add the action to destroy the preview when the PR is closed?

Comment (RinkiyaKeDad) [reply .github/workflows/preview.yaml:2]: Not that important from the article's perspective, so I skipped that. Do you want me to add it?

Comment (pchico83) [reply .github/workflows/preview.yaml:2]: every PR to this repo would leave preview ens running for 15 days no?

Comment (RinkiyaKeDad) [reply .github/workflows/preview.yaml:2]: Oh I forgot that would lead to this. Updated the PR!

**Pull request closed without merging**